plugin/__init__.py

```diff
@@ -8,19 +10,25 @@
 from .core.sessions import unregister_plugin
 from .core.types import ClientConfig
 from .core.types import LanguageConfig
+from .core.url import filename_to_uri
+from .core.url import uri_to_filename
 from .core.version import __version__
 
 # This is the public API for LSP-* packages
 __all__ = [
     '__version__',
     'AbstractPlugin',
     'ClientConfig',
+    'css',
+    'filename_to_uri',
     'LanguageConfig',
     'Notification',
+    'Promise',
```

Comment:
Should we really do that? For `on_pre_server_command` we could just expect an optional function to be returned and internally we would use a Promise.